compiler/definitions/ir/file_id.py

```diff
@@ -57,7 +60,8 @@ def to_ast(self):
         ##
         ## TODO: I am not sure about the FileDescriptor resource
         if(isinstance(self.resource, EphemeralResource)):
-            string = "{}#file{}".format(self.prefix, self.ident)
+            suffix = "{}#file{}".format(self.prefix, self.ident)
```

Comment:
Since these are FIFOs, can we name them something like `fifo`, `channel`, or `stream` rather than `file`?

(minor/I can do it)

Reply:
Good point :) I am doing it now!

Reply:
Done!